app/scripts/directives/logViewer.js

```diff
@@ -197,7 +194,8 @@ angular.module('openshiftConsole')
               var fill = $scope.fixedHeight ? $scope.fixedHeight : Math.floor($(window).height() - contentTop - pulserHeight);
               if (!$scope.chromeless && !$scope.fixedHeight) {
                 // Add some bottom margin if not chromeless.
```

Comment:
I cant think of a clever way to do it, but I wish we could share variable values between JS & LESS. Hmm.